### ISSUE TITLE
Soundfile option for "You joined channel"

### ISSUE THREAD

**Issue by Faxter** [mumble-voip/mumble#1966]
I like getting some form of audible feedback when working with audio-related tools like Mumble is.
That is the reason why I am missing some sound when I switch the channel. Something similar to the "Server Connected" message.

Request: Add an easy to configure sound option to the Settings -> Messages dialog for a "You joined channel" event.

**Comment by Kissaki** [comment]
Omicron [posted in the forum](https://forums.mumble.info/viewtopic.php?f=5&t=3356) requesting this as well, and pointing out that currently when you are being moved into a channel by someone else you do not get (audible) feedback. In that case it can't be argued that you are the action initiator, as is when joining a channel yourself.

**Comment by Ytmndissue** [comment]
This probably would not be too complicated to implement prior to the 1.3 release, you should only have to add two audio "hooks,":

One for notifying you when you join a channel.
One for notifying you when you are moved into a channel. 

Then you add two check boxes in the UI for the two cases above.

**Comment by Ytmndissue** [comment]
This should enable two checkboxes in the "messages" UI with the following labels:
You moved into channel
You were moved into channel

The first message is hooked into whenever you move yourself into another channel. This is an audible indication and confirmation similar to that of Ventrilo when you would enter a channel.

The second message is hooked into whenever someone else moves you into another channel. In other words, when the "mover" does not have the same user ID that you have.

A $25 bounty from Bountysource was posted for this, it is currently at $25.

**Comment by Ytmndissue** [comment]
A $25 bounty from Bountysource was posted for this, it is currently at $50.

**Comment by Ytmndissue** [comment]
Everything seems in order, once this is approved and has no issues (/merged into the master branch for 1.3) then this can get closed, and a claim can be made for the $50.

**Comment by davidebeatrici** [comment]
Pull request merged!

**Comment by mkrautz** [comment]
@davidebeatrici now we just need to fix the build to get a new snapshot released 😄

**Comment by davidebeatrici** [comment]
@mkrautz Indeed :wink:
